pkg/filters/data.go

```diff
@@ -11,28 +11,28 @@ import (
 	"github.com/aquasecurity/tracee/types/trace"
 )
 
-type ArgFilter struct {
+type DataFilter struct {
 	filters map[events.ID]map[string]Filter[*StringFilter]
 	enabled bool
 }
 
-// Compile-time check to ensure that ArgFilter implements the Cloner interface
-var _ utils.Cloner[*ArgFilter] = &ArgFilter{}
+// Compile-time check to ensure that DataFilter implements the Cloner interface
+var _ utils.Cloner[*DataFilter] = &DataFilter{}
 
-func NewArgFilter() *ArgFilter {
-	return &ArgFilter{
+func NewDataFilter() *DataFilter {
+	return &DataFilter{
 		filters: map[events.ID]map[string]Filter[*StringFilter]{},
 		enabled: false,
 	}
 }
 
-// GetEventFilters returns the argument filters map for a specific event
+// GetEventFilters returns the data filters map for a specific event
 // writing to the map may have unintentional consequences, avoid doing so
-func (af *ArgFilter) GetEventFilters(eventID events.ID) map[string]Filter[*StringFilter] {
+func (af *DataFilter) GetEventFilters(eventID events.ID) map[string]Filter[*StringFilter] {
 	return af.filters[eventID]
 }
 
-func (af *ArgFilter) Filter(eventID events.ID, args []trace.Argument) bool {
+func (af *DataFilter) Filter(eventID events.ID, datas []trace.Argument) bool {
```

Comment:
@josedonizetti this function uses `trace.Argument` as parameter... but I was in doubt if I need to change to `trace.Data`... I know you are working on the the new event struct.. so maybe you already change it

Reply:
1. I agree that changing trace.Argument should be part of the new event structure and not part of this PR
2. nit: `datas` should be `data` (which is already plural)